perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -288,6 +300,7 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
                  '-t {benchmark_name} -H {server_ip} -l {length} {confidence}'
                  ' -- '
                  '-P ,{{data_port}} '
+                 '-U {thinktime},{thinktime_array_size},{thinktime_run_length} '
```

Comment:
This will break running on machines that may have netperf already installed even if they don't want to use thinktime - can we not specify the -U arg unless users use the flags?